packages/contentful--cra-template-create-contentful-app/template/README.md

```diff
@@ -45,6 +45,33 @@ To make your app look and feel like Contentful use the following libraries:
 - [Forma 36](https://f36.contentful.com/) – Contentful's design system
 - [Contentful Field Editors](https://www.contentful.com/developers/docs/extensibility/field-editors/) – Contentful's field editor React components
 
+## Using the contentful-management SDK
```

Comment:
What about we do not introduce cma here and we add

```
npm i @contentful/contentful-management@latest @contentful/app-sdk@canary
```

in the instructions? The setup we provide won't work with app-sdk v3 and I wouldn't give the out-of-the-box option until app-sdk is stable.

Reply:
```suggestion
## Using the `contentful-management` SDK
```